<ui version="4.0" >
 <class>IconsOptionsWidgetClass</class>
 <widget class="QWidget" name="IconsOptionsWidgetClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>307</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>IconsOptionsWidget</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabDefault" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>466</width>
        <height>280</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Default iconset</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QListWidget" name="lwtDefaultIconset" >
         <property name="selectionMode" >
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="verticalScrollMode" >
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabContact" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>466</width>
        <height>280</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Default rules</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QTableWidget" name="twtDefaultRules" >
         <property name="editTriggers" >
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode" >
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="columnCount" >
          <number>2</number>
         </property>
         <column>
          <property name="text" >
           <string>RegExp</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Iconset</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabService" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>466</width>
        <height>280</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>User rules</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QTableWidget" name="twtUserRules" >
         <property name="selectionMode" >
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior" >
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="columnCount" >
          <number>2</number>
         </property>
         <column>
          <property name="text" >
           <string>RegExp</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Iconset</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pbtAddUserRule" >
           <property name="text" >
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtDeleteUserRule" >
           <property name="text" >
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
